R/provider-ollama.R

```diff
@@ -36,7 +42,8 @@ chat_ollama <- function(
   model,
   seed = NULL,
   api_args = list(),
-  echo = NULL
+  echo = NULL,
+  api_key = NULL
```

Comment:
I used `api_key = NULL` here intentionally -- as opposed to `api_key = ollama_key()` -- to signal that this is *not* required for most use cases.